src/vm-memory/src/mmap.rs

```diff
@@ -51,9 +59,36 @@ impl GuestRegionMmap {
         Ok(GuestRegionMmap {
             mapping,
             guest_base,
+            dirty_bitmap: None,
         })
     }
 
+    /// Provide the region with a dedicated bitmap to handle dirty page tracking.
+    pub fn enable_dirty_page_tracking(&mut self) {
+        let page_size = match unsafe { libc::sysconf(libc::_SC_PAGESIZE) } {
```

Comment:
had an offline discussion. Since sysconf can only return error when the parameter is not an accepted one (which will not ever be the case for us), I think it is ok to leave the expect